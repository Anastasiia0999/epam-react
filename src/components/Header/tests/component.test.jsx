import React from 'react';
import { mount} from 'enzyme';

import {Header} from "../index";

describe('<Header />', () =>{
    let wrapper;
    let instance;

    const props = {
        fields:{
            "logIn-btn" : "Log In",
            "logOut-btn" : "Log Out"
        },
        logStatus: false,
        englishOn: true
    };

    beforeEach(() => {
        wrapper = mount(<Header {...props} />);
        instance = wrapper.instance();
    });

    it('should render correctly', () =>{
        expect(wrapper.html()).toMatchSnapshot();
    });

});


