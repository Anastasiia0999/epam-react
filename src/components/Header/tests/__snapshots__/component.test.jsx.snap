// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Header /> should render correctly 1`] = `"<div class=\\"container\\"><div class=\\"row\\"><ul class=\\"card-header header-main col-12 my-3\\"><li><a class=\\"link-text\\" href=\\"/login\\"><h6 class=\\"link-text\\">Log In</h6></a></li><li class=\\"li-lang\\">Switch to Eng</li></ul></div></div>"`;
